.drone.star

```diff
@@ -2561,37 +2576,20 @@ def stopBuild():
     }]
 
 def uploadScreenshots():
-    return [{
-        "name": "upload-screenshots",
-        "image": "plugins/s3",
-        "pull": "if-not-exists",
-        "settings": {
-            "bucket": "owncloud",
-            "endpoint": {
-                "from_secret": "cache_s3_endpoint",
-            },
-            "path_style": True,
-            "source": "%s/tests/acceptance/reports/screenshots/**/*" % dir["web"],
-            "strip_prefix": "%s/tests/acceptance/reports/screenshots" % dir["web"],
-            "target": "/web/screenshots/${DRONE_BUILD_NUMBER}",
-        },
-        "environment": {
-            "AWS_ACCESS_KEY_ID": {
-                "from_secret": "cache_s3_access_key",
```

Comment:
is this env. still needed and should be updated or the access key from settings should just work :thinking:?